includes/functions.php

```diff
@@ -1006,6 +1006,11 @@ function get_enclosures( $post_id ) {
 
 	$enclosures = array_map(
 		function ( $enclosure ) {
+			// Check if the enclosure is a string.
+			if ( ! $enclosure || ! is_string( $enclosure ) ) {
+				return false;
```

Comment:
Should this return `null` to remove it from the array?

Reply:
Both will be filtered by `array_filter` but I am also fine with `null`.